### PR TITLE
Clean up examples 

### PR DESCRIPTION
**Pull request by danielpeintner**
fixes parts of https://github.com/eclipse/thingweb.node-wot/issues/654

### CODE REVIEW

**Comment by danielpeintner** [comment]
Even if this PR does not fix the full list of issues in https://github.com/eclipse/thingweb.node-wot/issues/654, I think it makes sense to review and approve it as is for now.
Otherwise it gets to BIG.

**Comment by danielpeintner** [comment]
@relu91 @JKRhb may I ask you to take a look?

File: examples/scripts/smart-coffee-machine-client.js [hunk 1]
```diff
@@ -17,46 +17,50 @@
...
         });
-        log(scheduledTask["message"], scheduledTask);
+        const scheduledTaskp = await scheduledTask.value();
+        log(scheduledTaskp.message, scheduledTaskp);
         // See how it has been added to the schedules property
-        let schedules = await thing.readProperty("schedules");
+        const schedules = await await (await thing.readProperty("schedules")).value();
         log("schedules value: ", schedules);
         // Let's set up a handler for outOfResource event
         thing.subscribeEvent("outOfResource", (data) => {
```
Comment (relu91) [inline examples/scripts/smart-coffee-machine-client.js:66]: *Warning* `data` is a `InteractionOutput` I would change the code of this callback to actually parse the event value (e.g. `log("outOfResource event",await data.value())`)

File: examples/scripts/smart-coffee-machine.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
- * Copyright (c) 2018 - 2020 Contributors to the Eclipse Foundation
+ * Copyright (c) 2018 - 2022 Contributors to the Eclipse Foundation
```
Comment (relu91) [inline examples/scripts/smart-coffee-machine.js:2]: I think we end up choosing just one year, not the year range right?

**Review by relu91 — APPROVED** [review]
Looks good so far! just some minor comments.

File: examples/scripts/smart-coffee-machine.js (hunk 2 above)
Comment (danielpeintner) [reply examples/scripts/smart-coffee-machine.js]: Good point, I fixed the files I touched in this PR. see https://github.com/eclipse/thingweb.node-wot/pull/669/commits/cc227af18d6188ae07dca1fdede2b600a5f17ced
I will do another run with the script to check copyright years for all files also.

File: examples/scripts/smart-coffee-machine-client.js (hunk 1 above)
Comment (danielpeintner) [reply examples/scripts/smart-coffee-machine-client.js]: Thanks for spotting this. Should be resolved by https://github.com/eclipse/thingweb.node-wot/pull/669/commits/c58f25259160b61ecdb6f7fcb1af71e731204877.

File: examples/scripts/smart-coffee-machine-client.js [hunk 3]
```diff
@@ -33,9 +33,9 @@ WoTHelpers.fetch("http://127.0.0.1:8080/smart-coffee-machine").then(async (td) =
         allAvailableResources = await (await thing.readProperty("allAvailableResources")).value();
         log("allAvailableResources value after change is:", allAvailableResources);
         // It's also possible to set a client-side handler for observable properties
-        /* thing.observeProperty("maintenanceNeeded", async (data) => {
+        thing.observeProperty("maintenanceNeeded", async (data) => {
             log("maintenanceNeeded property has changed! New value is:", data.value());
```
Comment (relu91) [inline examples/scripts/smart-coffee-machine-client.js:37]: Missing an await here

**Review by relu91 — CHANGES_REQUESTED** [review]
Minor.

File: examples/scripts/smart-coffee-machine-client.js (hunk 3 above)
Comment (danielpeintner) [reply examples/scripts/smart-coffee-machine-client.js]: 👍 good catch!
resolved by https://github.com/eclipse/thingweb.node-wot/pull/669/commits/7ddb0a1a2667d6fa30d2c0119609ddfebd1b6b5e

**Review by relu91 — APPROVED** [review]

**Pull request merged**